sakai-evaluation-tool/src/webapp/content/templates/modify_hierarchy_node_perms.html

```diff
@@ -113,19 +119,26 @@ <h3 rsf:id="msg=modifynodeperms.page.title">Hierarchy Node Perms</h3>
 						</td>
 						
 						<td class="actions">
-							<span id="addUserButton"><input type="submit" rsf:id="add-user-button" id="add-user-button" /></span>
+							<span class="saveButton"><input type="submit" rsf:id="save-changes-button" /></span>
+							<span class="removeUserButton"><input type="submit" rsf:id="remove-user-button" /></span>
 						</td>
 						
 					</tr>
 					
 				</tbody>
 				
 			</table>
-			
+
+			<!-- Clear changes and return to hierarchy UI buttons at the bottom of the form -->
+			<div class="action">
+				<span class="cancelButton">
+					<a href="#" rsf:id="return-link1">Return to Control Hierarchy</a>
+					<input type="submit" rsf:id="cancel-changes-button2" style="float: right;" />
```

Comment:
Same as above comment; should likely be addressed in a separate JIRA project wide.